index.html

```diff
@@ -5,47 +5,6 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
-    <link rel="stylesheet" href="./scss/index.css" />
-    <script src="./hamburger.js"></script>
   </head>
-  <body>
-    <header>
-      <div class="header_container">
```

Comment:
Its logo-container not header ....
you don't close header with some elements here in div.